content/en/docs/guides/input_and_access.md

```diff
@@ -202,6 +203,38 @@ Imports a helm chart from the local file system and adds it as a resource.
     input:
       type: helm
       path: ./megachart
+      repository: charts/mega
+```
+
+After transporting the corresponding component version to an OCI registry the helm chart will be made available under `charts/mega` prefixed by the name of the component version. This auto-prefix can be disabled by using a leading slash `/charts/mega`. If the `repository` tag is omitted the name of the helm chart from `Chart.yaml` will be used.
+
+It is also possible to import a helm chart from a helm chart repository:
+
+```yaml
+  resources:
+  - name: mariadb-chart
+    type: helmChart
+    input:
+      type: helm
+      helmRepository: https://charts.bitnami.com/bitnami
+      repository: charts/mariadb
+      path: mariadb
+      version: 12.2.7
+```
+
+Here the helm chart version `12.2.7` is copied from the path `mariadb` in helm chart repository `https://charts.bitnami.com/bitnami`. After transporting the corresponding component version to an OCI registry the helm chart will be made available under `charts/mariadb` prefixed by the name of the component version. This auto-prefix can be disabled by using a leading slash `/charts/mariadb`. If the `repository` tag is omitted the name of the helm chart from `Chart.yaml` will be used. There are additional optional fields `caCert` and `caCertFile` to specify a TLS certificate for the helm chart repository. The resulting path is contained in the field `referenceName` in the component descriptor:
+
+```yaml
+spec:
+  resources:
+      ...
+  - name: mariadb-chart
+    version: 12.2.7
+    type: helmChart
+    ...
+    access:
+      ...
+      referenceName: github.com/jensh007/mariadb/charts/mariadb:12.2.7
```

Comment:
It took me some time to figure out what is described here.
So this is how the resource looks like it has been transported to an OCI-Repository.
However, shouldn't it then look something like this?

```yaml
spec:
  resources:
      ...
  - name: mariadb-chart
    version: 12.2.7
    type: helmChart
    ...
    access:
      ...
      referenceName: ghcr.io/jensh007/github.com/open-component-model/megacomponent/charts/mariadb:12.2.7
```

Reply:
I will remove this paragraph. This section in the component-descriptor varies between the stages of transport. It is different between a cd in a CTF and in an OCI registry. So this is more confusing than helpful.